king_phisher/client/dialogs/campaign_selection.py

```diff
@@ -177,6 +219,11 @@ def signal_button_clicked(self, button):
 
 	def signal_checkbutton_toggled(self, _):
 		self._tv_model_filter.refilter()
+		self.gobjects['label_campaign_info'].set_text("Showing {0} of {1:,} campaign{2} found in the database.".format(
```

Comment:
Modify the string to remove "found in database." and capitalize the C in campaign.